examples/webserver/recipe/recipe.yaml

```
@@ -21,6 +21,8 @@ requirements:
   run:
     - python >=3.12
     - fastapi
+  run_constraints:
+    - fastapi >=0.110
```

```suggestion
    - pydantic >=2,<3
```

let's make it a more realistic scenario as fastapi is already in `run` (and update the test files)